cmake/Config.cmake.in

```diff
@@ -1,13 +1,8 @@
 #--------------------------- Base configuration for installation and rpath settings ---------------------------
 # Custom Install target, used in run target in source/CMakeLists.txt
-if (CMAKE_GENERATOR MATCHES "Makefiles")
-    # Make it multithreaded
-    add_custom_target( Install_${PROJECT_NAME} COMMAND "${CMAKE_COMMAND}" --build . --target
-            install -- -j WORKING_DIRECTORY "${CMAKE_BINARY_DIR}")
-else()
-    add_custom_target( Install_${PROJECT_NAME} COMMAND "${CMAKE_COMMAND}" --build . --target
-            install WORKING_DIRECTORY "${CMAKE_BINARY_DIR}")
-endif()
+add_custom_target( Install_${PROJECT_NAME} COMMAND "${CMAKE_COMMAND}" --build . --target
```

Comment:
don't we want o have `--parallel` here ?  [see doc](https://cmake.org/cmake/help/v3.18/manual/cmake.1.html#build-a-project)

Reply:
Well, I'm not sure of the doc... 
so if no -parallel, the use env var ? or if no number given then use env var ...

Reply:
Ok, I confirm, without -j or --parallel, the --build use the number of threads from env var.